Assets/Scripts/Interactables/Resource/Resource.cs

```diff
@@ -17,10 +18,20 @@ public class Resource : Interactable
         [SerializeField] protected int p_respawnTime;
 
         protected HealthData p_healthData;
+        protected bool p_hasStartedServer;
```

Comment:
```suggestion
        protected bool p_hasServerStarted;
```
Otherwise, it sounds like this starts the server